### DISCUSSION TITLE
Examples of videos using GFPGAN

### DISCUSSION THREAD

**Discussion by rebroad** [TencentARC/GFPGAN#358] [Show and tell]
I thought I'd start a threat of videos people might want to share showing GFPGAN's ability with videos.

Here's one I tried out with version 1.4 - it could be better but it could be worse - it makes the eyes a bit too dark, and doesn't cater well for blurred or fogged out faces - ideally it would use the context of the previous and later frames to know when to blur out the resulting face.

https://www.youtube.com/watch?v=GG6mdjdTgoo

**Comment by angelandy** [comment]
I also look for that ability, If I handle all the frames in video, sometime, face will  different,it will ok in image,but vedio will not so good

**Comment by Afreenrubel** [comment]
Hi
